backend/cfg/cfg_irc.ml

```diff
@@ -325,6 +325,7 @@ type direction =
   | Load_before_cell of Cfg.BasicInstructionList.cell
   | Store_after_cell of Cfg.BasicInstructionList.cell
   | Load_after_list of Cfg.BasicInstructionList.t
+  | Store_after_list of Cfg.BasicInstructionList.t
```

Comment:
@gretay-js After second look I believe everything is correct.

But currently loads are inserted `load 1, load 2, load 3, ...` into code but spills are inserted `..., store 3, store 2, store 1` for `Store_after_cell`
and `store 1, store 2, store 3, ...` for `Store_after_list`.

Would you oppose to changing it to `Store_before_list` in order to make it consistent with `Store_after_cell` (changing `Store_after_cell` is complicated)?

Reply:
Also `Store_after_list` only makes sense because the list is actually empty.
If we wanted to spill before a block which already has instruction (and we can because it has only one successor)
then `Store_before_list` would be required.